missingness/missingness.Rmd

```diff
@@ -0,0 +1,235 @@
+---
+title: "Arboviral Surveillance and Response Capacity Survey"
+author: "Missingness analysis"
+date: "`r format(Sys.time(), '%Y %B %d')`"
+output: html_document
+---
+
+```{r echo = FALSE, warning = FALSE, message = FALSE}
+
+# load the 2 packages needed:
+library(tidyverse)
+library(flextable)
+
+# process the survey results with our lightly customized version of the WHO script:
+source("survey_375147_R_syntax_file.R")
+
+# Iceland is for testing purposes only; remove for analysis:
+data <- data %>%
+  filter( SI01 != "Iceland")
+
+# load data dictionary to facilitate processing results:
+dict <- read_csv("Data_dictionary_Survey375147.csv")
```

Comment:
I get an error here:

```> dict <- read_csv("Data_dictionary_Survey375147.csv")
Error: 'Data_dictionary_Survey375147.csv' does not exist in current working directory ('/home/joebrew/Documents/arbovirus/missingness').
```

Reply:
@mbrudd I resolved this by copying this file from the `misc` folder to here. But there should be instructions to do this...

Reply:
same comment as above re: `instructions.md`